src/main/resources/liquibase/crop_changelog/21_3_0.xml

```diff
@@ -0,0 +1,773 @@
+        <preConditions onFail="MARK_RAN">
+			<columnExists tableName="stock" columnName="value"/>
+		</preConditions>
+		<comment>Rename column 'value' to 'value_bkp' in table 'stock'</comment>
+		<sql>
+			ALTER TABLE stock CHANGE value value_bkp varchar(255) null;
+		</sql>
+	</changeSet>
+
+	<changeSet author="lkovacic" id="v21.3.0-31">
+        <preConditions onFail="MARK_RAN">
+			<columnExists tableName="stock" columnName="type_id"/>
+		</preConditions>
+		<comment>Rename column 'type_id' to 'type_id_bkp' in table 'stock' and make it nullable</comment>
+		<sql>
+			ALTER TABLE stock CHANGE type_id type_id_bkp int null;
+		</sql>
+	</changeSet>
+
+	<changeSet author="cuenyad" id="v21.3.0-32">
+		<preConditions onFail="MARK_RAN">
+			<columnExists tableName="listdata" columnName="entrycd"/>
+		</preConditions>
+		<comment>Rename column 'entrycd' to 'entrycd_bkp' in table listdata and make it nullable.</comment>
+		<sql dbms="mysql" splitStatements="true">
+		ALTER TABLE listdata
+			CHANGE COLUMN entrycd entrycd_bkp VARCHAR (47) NULL DEFAULT '-';
+		</sql>
+	</changeSet>
+	<!-- End of ENTRY_CODE related changesets -->
+
+	<changeSet author="cuenyad" id="v21.3.0-33">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT COUNT(*) > 0 FROM CVTERM WHERE IS_SYSTEM = 1
+					AND CVTERM_ID IN (8377, 8250, 8240, 8330, 8340, 8235, 8378, 8201);
+            </sqlCheck>
+        </preConditions>
+        <comment>Mark Germplasm Descriptors as system variables</comment>
+        <sql dbms="mysql" splitStatements="true">
+			 UPDATE CVTERM SET IS_SYSTEM = 1
+			 WHERE CVTERM_ID IN (8377, 8250, 8240, 8330, 8340, 8235, 8378, 8201);
+        </sql>
+	</changeSet>
+
+	<changeSet author="lkovacic" id="v21.3.0-34">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT count(1) > 0 FROM projectprop WHERE type_id = 1804 AND variable_id NOT IN (8377, 8250, 8240, 8340, 8330, 8201, 8378, 8235);
+			</sqlCheck>
+		</preConditions>
+		<comment>Remove crop specific and created by user germplasm descriptors from 'projectprop' table</comment>
+		<sql dbms="mysql" splitStatements="true">
+			DELETE FROM projectprop WHERE type_id = 1804 AND variable_id NOT IN (8377, 8250, 8240, 8340, 8330, 8201, 8378, 8235);
+		</sql>
+	</changeSet>
+
+	<!-- GERM_SOURCE related changesets -->
+	<changeSet author="lkovacic" id="v21.3.0-35">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT count(1) FROM cvtermprop WHERE cvterm_id = 8320 AND type_id = 1800 AND value = 'Germplasm Descriptor';
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			Remove 'Germplasm Descriptor' type to GERM_SOURCE
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			DELETE FROM cvtermprop WHERE cvterm_id = 8320 AND type_id = 1800 AND value = 'Germplasm Descriptor';
+
+			DELETE FROM stockprop WHERE type_id = 8320;
+			DELETE FROM projectprop WHERE variable_id = 8320 AND type_id = 1804;
```

Comment:
@lik-83 delete from projectprop not needed, it was covered in v21.3.0-34

Reply:
Done